ui-mockups/src/css/style.css

```diff
@@ -27,7 +27,7 @@ small a:hover,
 .ah-author-link:hover {
   text-decoration: none;
   color: #000000;
-  background-color: #FFDE00; }
```

Comment:
The background color here was fine. Changing the font size makes it very dramatic